http-service-lambda/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["Kan-Ru Chen <kanru@kanru.info>"]
```

Comment:
IMO, it makes sense to use `authors` from `http-service-hyper` and other crates here, too. (Also: I'd apply for membership in the GitHub organization.)

Reply:
Hi, I tried to implement this but I think I don't fully understand what you mean. It seems each crate has different `authors` set and it doesn't make sense to make a union of them here.

I'd love to apply for membership. Where/who can I put such application?

Reply:
Sorry I'm just now getting back to this.

[Here's an example of how the `authors` line in your `Cargo.toml` should look](https://github.com/rustasync/http-service/blob/master/http-service-hyper/Cargo.toml#L2-L5). It's cool to substitute your name and email for the last entry in my example, but I think it makes sense for maintainers to be able to publish this package alongside others in the repo.

As for applying for rustasync membership, you can ask in Discord (link on the [org page](https://github.com/rustasync)). I don't hang out much there anymore, but plenty of other folks do.

Reply:
I went ahead and resolved this while I was working on other things this evening. Hope you don't mind.

Reply:
No problem. Thanks for updating the PR!